tests/testsuite.src/run_misc.at

```diff
@@ -15251,3 +15251,324 @@ AT_CHECK([$COMPILE prog.cob])
 AT_CHECK([$COBCRUN_DIRECT ./prog], [0], [1.2345E-5 1.2345E-5], [])
 
 AT_CLEANUP
+
+
+AT_SETUP([profiling and codegen])
+AT_KEYWORDS([EVALUATE])
```

Comment:
Shouldn't that be `AT_SETUP([profiling EVALUATE])`?
... and _possibly_ most of those test include the keyword runmisc?